activation/atxdb.go

```diff
@@ -201,8 +181,13 @@ func (db *ActivationDb) ValidateAtx(atx *types.ActivationTx) error {
 		if !posAtx.Valid {
 			return fmt.Errorf("positioning atx is not valid")
 		}
+		if atx.PubLayerIdx <= posAtx.PubLayerIdx {
```

Comment:
why `=` ?

Reply:
I wrote this to you in slack - I don't think a posAtx on the same layer should be considered valid. How can you publish something that references something on the same layer?

It's unlikely to be valid even if it's very close because you have to start and finish PoET after seeing the posATX, but it's hard to set a clear boundary. Publishing it on the same layer, though, is definitely not according to the rules.